src/main/java/uk/gov/companieshouse/acsp/controllers/AcspController.java

```diff
@@ -39,7 +43,14 @@ public ResponseEntity<Object> getAcspData(@PathVariable(TRANSACTION_ID_KEY) Stri
                                               @PathVariable("id") String id,
                                               @RequestHeader(value = ERIC_ACCESS_TOKEN) String requestId) {
         LOGGER.info("received request to get acsp data");
-        var acspData = acspService.getAcsp(id);
+        Transaction transaction;
```

Comment:
You can declare this inside the try block only using var

Reply:
moved it to inside try catch block. Kept the Type same as we know the return type will be transaction.

Reply:
I think this adds a code smell.